tests/test_compute.py

```diff
@@ -276,14 +281,16 @@ def test_compute(client, publisher_wallet, consumer_wallet):
 
 @pytest.mark.integration
 def test_compute_diff_provider(client, publisher_wallet, consumer_wallet):
-    environments = get_c2d_environments()
+    environment = get_free_c2d_env()
```

Comment:
can we add this as a fixture in conftest.py instead?

Reply:
feel free to do it :)